app/models/physical_server.rb

```diff
@@ -6,6 +6,7 @@ class PhysicalServer < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::PhysicalInfraManager"
 
   default_value_for :enabled, true
+  has_one :host, :foreign_key => "service_tag", :primary_key => "serial_number"
```

Comment:
The names `has_one` and `belongs_to` are merely reflecting where the foreign key lives.  If we want to move the foreign key over to the `hosts` table, then the model verbiage would reverse.

See also:

http://guides.rubyonrails.org/association_basics.html#the-belongs-to-association

http://guides.rubyonrails.org/association_basics.html#the-has-one-association